app/api/v2/views/redflag_views.py

```diff
@@ -29,17 +29,17 @@ def post(self):
  filled all fields. i.e {} " .format(data)}]
             }), 404)
 
-        videos = data['videos']
         comment = data['comment']
-        images = data['images']
         location = data['location']
+        images = data['images']
```

Comment:
Are your images and videos string or they should be files that can be uploaded?

Reply:
My image fields are image and video strings.